cmd/fluxd/main.go

```diff
@@ -232,17 +253,57 @@ func main() {
 		os.Exit(0)
 	}
 
+	viper.SetConfigName(config.ConfigName)
+	viper.AddConfigPath(config.ConfigPath)
+
```

Comment:
```suggestion
viper.SetConfigType("foo")
```

once/if we settle on a single format

Reply:
This method was intended to be for the situation in which the config file has no extension -- but, I have checked, and it turns out that you can set the config file name to e.g., "flux-config.yaml" and set the type, to get the effect desired here.